package myexample

object A {
    // new scala.util.control.Exception.Catch()
}
